web/src/main/java/org/cbioportal/web/parameter/sort/ClinicalAttributeSortBy.java

```diff
@@ -8,7 +8,8 @@
     datatype("datatype"),
     patientAttribute("patientAttribute"),
     priority("priority"),
-    studyId("cancerStudyIdentifier");
+    studyId("cancerStudyIdentifier"),
+	clinicalAttributeCount("attrCount");
```

Comment:
Indentation, and the name `clinicalAttributeCount` doesn't match with the property name. These two are supposed to be different only if the alias in the SQL query is different than what we return to the user.